QUARKUS-3004.md

```diff
@@ -0,0 +1,33 @@
+# QUARKUS-3004: ARM support for JVM mode on OCP
+
+Jira: https://issues.redhat.com/browse/QUARKUS-3004
+
+The goal of this feature is to provide production support for Red Hat build of Quarkus running in JVM mode on OpenShift
+on most commonly used server side 64-bit ARM architectures.
+
+## Scope of the testing
+To verify that RHBQ 3 properly functions on ARM achitectures in JVM mode, the OpenShift test suite will have to pass 
+with OpenJDK 17.
+
+### Impact on test suites and testing automation
+* OCP testings
+  * OCP4 IPI install/destroy on AWS on ARM job will have to be implemented
+  * A pipeline will have to be implemented in extended platforms that will setup OCP on ARM, run Quarkus QE OpenShift 
+    testsuite on it and then clean it up
+  * Determine which of operators and testing services are supported on OCP on ARM
```

Comment:
I agree this  must be done, but I'm not sure it it belongs to _Impact on test suites and testing automation_, can you add a hint how it is going to be determined - e.g. run all tests and find issues, then analyze them and let PM to determine if we are going to support failing cases too (and fix them..), or there will be JIRA that lists supported features (support scope)?

Reply:
So @mjurc you are expecting that we end up with aarch64 specific profile in our TS?

Reply:
@michalvavrik I've clarified the wording - hopefully this is better? I've named the impact on test suite in followup points and thus also addressed @rsvoboda 's concern.